app/views/providers/means/vehicle_details/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= form_with(
```

Comment:
Validation: to endure error summary correctly links to the first radio button, do we need set the option `link_errors` to `true`?

Reply:
Only the payments_remain needed one.  The govuk helpers handle the others, e.g. govuk_text_field links automatically :+1: